version: "3.7"

services:
  jupyterlab-server:
    image: kaustvl/introduction-to-python-for-data-science:fall-2019
    ports:
      - "8888:8888"
    volumes:
      - ../bin:/home/al-khawarizmi/app/bin
      - ../data:/home/al-khawarizmi/app/data
      - ../data:/home/al-khawarizmi/app/notebooks
      - ../src:/home/al-khawarizmi/app/src
    init: true
    stdin_open: true
    tty: true    
